src/Controls/src/Core/Shell/Shell.cs

```diff
@@ -80,10 +97,20 @@ static void OnSearchHandlerPropertyChanged(BindableObject bindable, object oldVa
 				SetInheritedBindingContext(newHandler, bindable.BindingContext);
 		}
 
-		/// <summary>Bindable property for attached property <c>FlyoutItemIsVisible</c>.</summary>
+		/// <summary>
+		/// The <see cref = "FlyoutItem" /> visibility.
+		/// Flyout items are visible in the flyout by default.
+		/// </summary>
 		public static readonly BindableProperty FlyoutItemIsVisibleProperty =
 			BindableProperty.CreateAttached("FlyoutItemIsVisible", typeof(bool), typeof(Shell), true, propertyChanged: OnFlyoutItemIsVisibleChanged);
 		public static bool GetFlyoutItemIsVisible(BindableObject obj) => (bool)obj.GetValue(FlyoutItemIsVisibleProperty);
+
+		/// <summary>
+		/// Sets the visibility of <see cref = "FlyoutItems" />.
+		/// Flyout items are visible in the flyout by default. However, an item can be hidden in the flyout with the FlyoutItemIsVisible property.
+		/// </summary>
+		/// <param name="obj">The object that sets the visibility of flyout items.</param>
```

Comment:
This a bit ambiguous as to whether `obj` is in charge of setting the visibility, or whether the value of  the `FlyoutItemIsVisibleProperty` is set _within_ the object. 

Not sure if this comment fully makes sense. @PureWeen  could you double check this conveys the right message? This pattern is common throughout shell, so this might require other documents added in this PR, but I think its worthwhile for the customers.

Reply:
Hmm, so this is a typical structure for any of our attached properties. Do we have typical language we use for attached properties?

This is similar
https://github.com/dotnet/maui/blob/c7d1a4ec8d857aba674362d2777d98855f0ca67a/src/Controls/src/Core/NavigationPage/NavigationPage.cs#L340

https://learn.microsoft.com/en-us/uwp/api/windows.ui.xaml.controls.grid.row?view=winrt-22621

"Sets if this obj should have a visible flyout item in shells flyout menu" ?

Reply:
Updated using Shane suggestion just adding crefs.